fvm/env.go

```diff
@@ -572,7 +571,7 @@ func (e *hostEnv) VerifySignature(
 		defer sp.Finish()
 	}
 
-	valid, err := verifySignatureFromRuntime(
+	_, err := verifySignatureFromRuntime(
```

Comment:
This looks good. But I wanted to make sure this is not the same function that is called  in this [contract](https://github.com/onflow/cadence/blob/master/runtime/stdlib/contracts/crypto.cdc#L147), otherwise we would be introducing another breaking change.

Reply:
Right, I missed that: This should stay as-is, i.e. verification for existing signature algorithm / hash algorithm pairs should keep working, only new ones show always return false

Reply:
`verifySignatureFromRuntime` already returns an error for the new hash/sig algo so I reverted this back to what it was.